src/routes/healthcheck.ts

```
@@ -0,0 +1,88 @@
+      checks.services.xmtp.status = "unhealthy";
+      checks.services.xmtp.error = xmtpHealth.error || "Unknown XMTP error";
+      checks.services.xmtp.environment = xmtpHealth.environment || null;
+      checks.services.xmtp.customHost = xmtpHealth.customHost || null;
+      overallStatus = 503;
+      logger.error("XMTP health check failed:", xmtpHealth.error);
+    }
+
+    // Check notification service connectivity (optional, only if configured)
+    try {
+      if (process.env.NOTIFICATION_SERVER_URL) {
+        const notificationClient = createNotificationClient();
```

ESLint

[error] Delete `··`

(prettier/prettier)